deploy/provision/kvm-mullvad-upstream.sh

```
@@ -0,0 +1,154 @@
+    --prio)   PRIO="${2:-}"; shift 2 ;;
+    --dry-run) DRY=1; shift ;;
+    --down)   DOWN=1; shift ;;
+    -h|--help) sed -n '2,45p' "$0"; exit 0 ;;
+    *) err "unknown arg: $1"; exit 2 ;;
+  esac
+done
+
+[ "$(id -u)" = "0" ] || { err "must run as root"; exit 1; }
+
+run() { # execute or (dry-run) print
+  if [ "$DRY" = "1" ]; then echo -e "  ${YELLOW}\$${NC} $*"; else eval "$@"; fi
```

Shellcheck (0.11.0)

[warning] eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string).

(SC2294)